Objects/StrategyCardAutomator.ttslua

```diff
@@ -915,7 +814,16 @@ function updateUiOnClick(player, option, alt)
 
 	local layout = UI.getXmlTable()
 
-	-- adds a tech primary and secondary to automatically consume the stategy token
+	-- save and reset visibility for default layout
+	if not _defaultLayoutBackup then
+		_defaultLayoutBackup = _deepcopy(layout)
+		for _,menu in ipairs(_defaultLayoutBackup) do
+			menu['attributes']['visibility'] = ""
+			menu['attributes']['active'] = "false"
+		end
+	end
+
+	-- adds the default "Primary, Secondary, Pass, Close" menu moves the research menu to technologyPopup
```

Comment:
Save the original UI layout and reset visibility (to prevent UI from opening in onObjectEnterContainer())